app/utils/auth.py

```
@@ -0,0 +1,277 @@
+                    log_message=INVALID_API_KEY
+                )
+
+            # Check if API key is active (assuming you have this field)
+            if hasattr(api_key_record, 'is_active') and not api_key_record.is_active:
+                raise UnauthorizedAccess(
+                    log_message=API_KEY_IS_INACTIVE
+                )
+            user_record = await self.user_store.get_by_user_id(api_key_record.user_id)
+            if not user_record:
+                raise UnauthorizedAccess(
+                    log_message=f"User not found"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)